website/src/components/views/CreateJob.jsx

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import queryString from 'query-string'
+import { NavBar } from '../NavBar';
+
+class CreateJob extends React.Component {
+    constructor(props) {
+        super(props);
+        this.onSubmitWithProps = this.onSubmitWithProps.bind(this)
```

Comment:
You can delete this line if when you define this function later on, you define it using fat-arrow syntax instead i.e.

```jsx
onSubmitWithProps = async (event, props) => {
   // elided code...
}
```